src/main/docs/guide/breaks.adoc

```diff
@@ -2,23 +2,4 @@ This section documents breaking changes between Micronaut Neo4J versions:
 
 === Micronaut Neo4J {version}
```

Comment:
I know this has been here before, but it's weird...

I think we normally keep the old breaks, and use a major release version instead of some weird variable

https://github.com/micronaut-projects/micronaut-views/blob/master/src/main/docs/guide/breaks.adoc